.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,66 @@
+---
+name: Bug report
+about: Something isn't working as expected
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!-- NOTE:
+
+If you find that markdown files are not rendered as expected, please go to https://marked.js.org/demo/ to see if it can be reproduced there. If it can be reproduced, please file a bug to https://github.com/markedjs/marked.
+
+If you want help on your bug, please also send us the git repository (GitHub, GitLab, Bitbucket etc...) where your hexo code is stored. It would greatly help.
```

Comment:
suggest to add, "If you prefer not to have your hexo code out in public, please upload to a private GitHub repository and grant read-only access to hexojs/core".

Reply:
I will update later :)

Reply:
Done :)